rail_scripts/rail-preprocess-parquet

```diff
@@ -92,31 +93,36 @@ def build_output_name(file_name, index, template, extension=DEFAULT_EXT):
 
     return template.format(fname=split[0], idx=index, ext=ext)
 
-def parquet_check_column_presence(parquet, flux, mag, ra, dec):
+def parquet_check_column_presence(parquet, flux, mag, ra, dec, objectId):
     columns = frozenset(parquet.schema.names)
     has_magnitude = all(col in columns for col in mag)
     has_flux = all(col in columns for col in flux)
     has_ra = ra in columns
     has_dec = dec in columns
+    has_objectId = objectId in columns
```

Comment:
Inconsistent coding style: please use underscore notation for variable names: `has_object_id`.